### PR TITLE
Homework 4 task 3

### PR DESCRIPTION
**Pull request by ivanzuev78**
Homework 4 task 3 done

### CODE REVIEW

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
This task is about tests.  Do we have any special cases?

**Comment by ivanzuev78** [comment]
> This task is about tests. Do we have any special cases?

Did you mean wrong input?

**Comment by Cjkjvfnby** [comment]
It is not wrong (type str), it is special.

File: HomeWork_4/test_task_3.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def test_my_precious_logger_stderr(capsys):
+
+    my_precious_logger("error: Force not found")
+    captured = capsys.readouterr()
+    assert captured.err == "error: Force not found"
+
+
+def test_my_precious_logger_wrong_input():
+
+    with pytest.raises(
```
Comment (Cjkjvfnby) [inline HomeWork_4/test_task_3.py:22]: This test just checks how Python works. 

 If checking the argument type is not in requirements of function then there is no need to add a test for it.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

**Comment by ivanzuev78** [comment]
The only special case I found was empty input.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
Please for each test check that other output is empty.

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**